crates/apub/src/api/user_settings_backup.rs

```diff
@@ -377,13 +377,14 @@ mod tests {
     import_settings(backup, import_user.clone(), context.reset_request_count())
       .await
       .unwrap();
-    let import_user_updated = LocalUserView::read(&mut context.pool(), import_user.local_user.id)
-      .await
-      .unwrap();
 
     // wait for background task to finish
     sleep(Duration::from_millis(100)).await;
```

Comment:
Try increasing this time.

Reply:
The checks passed on this CI below. You're seeing that locally?

Reply:
It also passed when the PR was merged and then failed later on main branch. Probably depends if CI runs on a fast or slow builder which would change the delay needed. I dont see how moving this line would change anything. And no I dont get it locally, probably because my computer is too fast.

Reply:
You need to fetch the updated data *after* waiting for the background stuff to update. The wait you performed was pointless because it fetched it before waiting.

Reply:
Looks like you are fixing a different error from the one I saw. Makes sense then.